frontend/src/components/LandingPage/Explanation/Explanation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Jumbotron } from 'reactstrap';
+
+const Explanation = () => {
```

Comment:
What do you think about changing this to a functional component instead of returning an arrow function?

Reply:
Good point! I have updated this in the latest commit.